ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/community_lists.schema.yml

```
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  community_lists:
+    type: list
+    primary_key: name
+    convert_types:
+      - dict
+    display_name: Community Lists (legacy model)
+    description: |
+      AVD supports 2 different data models for community lists:
+
+      - The legacy `community_lists` data model that can be used for compatibility with the existing deployments.
+      - The improved `ip_community_lists` data model.
+
+      Both data models can coexist without conflicts, as different keys are used: `community_lists` vs `ip_community_lists`.
+      Community list names must be unique.
+
+      The legacy data model supports simplified community list definition that only allows a single action to be defined as string:
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+          display_name: Community-list Name
+        action:
+          type: str
+          required: true
+          description: |
+            Action as string
+            Example: "permit any any"
```

I am not sure this example is correct for a community list.
```suggestion
            Example: "permit GSHUT 65123:123"
```